app/frontend/packs/UserEditPack.res

```diff
@@ -13,35 +14,36 @@ let decodeProps = json => {
   )
 }
 
-Psj.match("users#edit", () => {
-  let (
-    name,
-    about,
-    locale,
-    availableLocales,
-    avatarUrl,
-    dailyDigest,
-    hasCurrentPassword,
-    isSchoolAdmin,
-    hasValidDeleteAccountToken,
-  ) =
-    DomUtils.parseJSONTag(~id="user-edit__props", ()) |> decodeProps
-  switch ReactDOM.querySelector("#react-root") {
-  | Some(root) =>
-    ReactDOM.render(
-      <UserEdit
-        name
-        about
-        locale
-        availableLocales
-        avatarUrl
-        dailyDigest
-        hasCurrentPassword
-        isSchoolAdmin
-        hasValidDeleteAccountToken
-      />,
-      root,
-    )
-  | None => ()
-  }
-})
+let (
```

Comment:
@developedBySJ Why was the Page-Specific JavaScript handler removed from this pack file? 😕

Reply:
Accidently removed it when I was merging the `master` branch.
I will update code.